.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -13,10 +13,10 @@ assignees: ''
 # Steps to reproduce
 ...
 
-Expected behaviour
+# Expected behaviour
 ...
 
-# Images and documents
+# Documents and images
```

Comment:
This PR contains changes that are already in main. I suspect the reason is you branched off of main instead of develop. No worries at all, this happened to Massimo and myself. The fix is to rebase this branch onto develop. We can get on a call and fix this, also a great git learning opportunity.